lib/galaxy/tools/loader_directory.py

```diff
@@ -22,6 +22,7 @@
 
 YAML_EXTENSIONS = [".yaml", ".yml", ".json"]
 CWL_EXTENSIONS = YAML_EXTENSIONS + [".cwl"]
+EXCLUDE_WALK_DIRS = ['.hg', '.git']
```

Comment:
we could also exclude `.venv` here, right ?

Reply:
It doesn't hurt? But probably not necessary. I don't expect .venvs in the shed tool directories, but again won't hurt anything.

Reply:
Ah, I didn't get that this applied to tool directories only (it even is in the file name), but yeah, I guess it doesn't hurt.